### PR TITLE
Fix linux build not package java jars

### PR DESCRIPTION
**Pull request by ConeyLiu**
## Why are these changes needed?
Fix linux build not package java jars.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by ConeyLiu** [comment]
Hi @chaokunyang, could you help to review this?

File: python/build-wheel-manylinux2014.sh [hunk 1]
```diff
@@ -41,11 +41,6 @@ export JAVA_HOME="$java_home"
 #echo "build --config=manylinux2010" >> /root/.bazelrc
 echo "build --incompatible_linkopts_to_linklibs" >> /root/.bazelrc
 
-if [[ -n "${RAY_INSTALL_JAVA:-}" ]]; then
-  bazel build //java:ray_java_pkg
-  unset RAY_INSTALL_JAVA
-fi
```
Comment (ConeyLiu) [inline python/build-wheel-manylinux2014.sh:47]: `python setup.py bdist_wheel` should trigger the `bazel build //java:ray_java_pkg`

Comment (chaokunyang) [reply python/build-wheel-manylinux2014.sh:47]: `bazel build //java:ray_java_pkg` has been executed here, so it's not necessary to execute it in `python setup.py bdist_wheel`.

Comment (chaokunyang) [reply python/build-wheel-manylinux2014.sh:47]: execute `bazel build //java:ray_java_pkg` in `python setup.py bdist_wheel` will execute three times for python 3.6, 3.7, 3.8

Comment (chaokunyang) [reply python/build-wheel-manylinux2014.sh:47]: But if we don't build java in python setup.py, the jars will not be copied in the python build process. This is a bug

Comment (ConeyLiu) [reply python/build-wheel-manylinux2014.sh:47]: yeah, if we unset the `RAY_INSTALL_JAVA` the jars will not be copied to whl.

Comment (chaokunyang) [reply python/build-wheel-manylinux2014.sh:47]: I submitted a PR, could you help reivew it? #12456

**Pull request closed without merging**